Add section about improving load times (#28)
elastic/universal-profiling-documentation: README.md

To enable debug logs, add the `-verbose` command-line flag or the `verbose true` setting in the configuration file.

### Improve load times

> The amount of data to be loaded for the flamegraph, the topN functions and the traces view is often several megabytes.

Reviewer: ```suggestion
The amount of data loaded for the flamegraph, topN functions, and traces view can lead to latency when using a slow connection (e.g. DSL or mobile).
```

> This can lead to an annoying latency when acting on a slow connection (e.g. DSL or mobile).

Reviewer: ```suggestion

```

If you already run Kibana/ES 8.6+, setting the Kibana cluster option `server.compression.brotli.enabled: true` reduces

> the amount of data to be transferred and thus it should reduce your time to wait.

Reviewer: ```suggestion
the amount of data to be transferred and should reduce load time.
```
